framework/src/org/apache/cordova/CordovaActivity.java

```diff
@@ -115,8 +116,24 @@ public void onCreate(Bundle savedInstanceState) {
             getWindow().setFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN,
```

Comment:
SetFullscreen is deprecated, but we shouldn't make it behave differently from "Fullscreen". How about changing this to:

    if (SetFullscreen) {
      Log();
      preferences.set("Fullscreen", true);
    }
    if (Fullscreen) {
      ...
    }